### PR TITLE
Switch id to login in logger statements

### PR DESCRIPTION
**Pull request by Mathieu414**

### CODE REVIEW

File: app/pages/tokens/send_reset_password.php [hunk 1]
```diff
@@ -17,11 +17,11 @@
             ->createEmail($user->real_email, $subject, $content)
             ->send();
         if ($result->success) {
-            logger()->info("User $user->id password reset email sent");
+            logger()->info("User $user->login password reset email sent");
```
Comment (Atmos4) [inline app/pages/tokens/send_reset_password.php:20]: Add login inside the context here, with PSR3 formatting
```suggestion
            logger()->info("User {login} password reset email sent", ["login" => $user->login]);
```

File: app/pages/tokens/send_reset_password.php [hunk 2]
```diff
@@ -17,11 +17,11 @@
...
             $v->set_success("Mail envoyé à l'adresse " . MailHelper::obfuscate($user->real_email));
             em()->flush();
         } else {
-            logger()->warning("User $user->id password reset email failed to send");
+            logger()->warning("User $user->login password reset email failed to send");
```
Comment (Atmos4) [inline app/pages/tokens/send_reset_password.php:24]: Same as above comment

File: app/pages/tokens/user_activation.php [hunk 3]
```diff
@@ -18,7 +18,7 @@
 $phone = $v->phone("phone")->label("Numéro de téléphone");
 
 if ($v->valid()) {
-    logger()->info("User {$token->user->id} activated");
+    logger()->info("User {$token->user->login} activated");
```
Comment (Atmos4) [inline app/pages/tokens/user_activation.php:21]: Same as above

File: app/pages/users/user_add.php [hunk 4]
```diff
@@ -34,7 +34,7 @@
         em()->persist($token);
         em()->persist($new_user);
         em()->flush();
-        logger()->info("User {$new_user->id} created and activation email sent");
+        logger()->info("User {$new_user->login} created and activation email sent");
```
Comment (Atmos4) [inline app/pages/users/user_add.php:37]: Same as above

File: app/pages/users/user_delete_confirm.php [hunk 5]
```diff
@@ -12,7 +12,7 @@
     $form->set_error("Impossible de supprimer un utilisateur actif");
 }
 if ($form->valid()) {
-    logger()->info("User {$user->id} deleted by user " . User::getCurrent()->id);
+    logger()->info("User {$user->login} deleted by user " . User::getCurrent()->id);
```
Comment (Atmos4) [inline app/pages/users/user_delete_confirm.php:15]: Same as above

File: app/services/OvhService.php [hunk 6]
```diff
@@ -34,7 +34,7 @@ function deactivateUser(User $user)
                 if ($this->ovhClient->getMailingListSubscriber($list, $user->real_email)) {
                     try {
                         $this->ovhClient->removeSubscriberFromMailingList($list, $user->real_email);
-                        logger()->info("User {$user->id} removed from mailing list {$list})");
+                        logger()->info("User {$user->login} removed from mailing list {$list})");
```
Comment (Atmos4) [inline app/services/OvhService.php:37]: Same as above

**Review by Atmos4 — DISMISSED** [review]

File: app/pages/tokens/send_reset_password.php (hunk 1 above)
Comment (Mathieu414) [reply app/pages/tokens/send_reset_password.php:20]: ✅

File: app/pages/tokens/send_reset_password.php (hunk 2 above)
Comment (Mathieu414) [reply app/pages/tokens/send_reset_password.php:24]: ✅

File: app/pages/tokens/user_activation.php (hunk 3 above)
Comment (Mathieu414) [reply app/pages/tokens/user_activation.php:21]: ✅

File: app/pages/users/user_add.php (hunk 4 above)
Comment (Mathieu414) [reply app/pages/users/user_add.php:37]: ✅

File: app/pages/users/user_delete_confirm.php (hunk 5 above)
Comment (Mathieu414) [reply app/pages/users/user_delete_confirm.php:15]: ✅

File: app/services/OvhService.php (hunk 6 above)
Comment (Mathieu414) [reply app/services/OvhService.php:37]: ✅

**Review by Atmos4 — APPROVED** [review]

**Pull request merged**